src/platforms/macos/macosutils.mm

```diff
@@ -190,11 +190,9 @@ @implementation NSStatusBarButton (Swizzle)
 - (void)setImagePatched:(NSImage*)image {
   NSImage* img = image;
 
-  if (@available(macOS 11.0, *)) {
```

Comment:
Can you tell me more what happens on MacOS <11? Is the API available? And why we had this it-stmt?

Reply:
The API is available on `< 11`. The if-stmt — and the entire function was added to fix a Qt bug that caused status bar icons to be displayed stretched. The idea behind removing this statement is: Now this function resizes the image always to fit the size of the status bar so we can use one image for retina and non-retina displays. Which would be necessary now we use `png` instead of `svg` graphics.